core/config/v2/types.go

```diff
@@ -50,9 +50,21 @@ type Core struct {
 
 type Secrets struct {
 	DatabaseURL       *models.URL
+	DatabaseBackupURL *models.URL
+
 	ExplorerAccessKey *string
 	ExplorerSecret    *string
-	//TODO https://app.shortcut.com/chainlinklabs/story/33624/add-secrets-toml
+
+	KeystorePassword *string
+	VRFPassword      *string
+}
+
+func (s *Secrets) String() string {
+	return "<hidden>"
+}
```

Comment:
Worth also implementing a `GoString` and forcing conformity to `GoStringer` interface